cli/src/main.rs

```diff
@@ -51,33 +87,51 @@ async fn main() -> std::io::Result<()> {
       repl::main();
       Ok(())
     }
-    Cli::Show { input, typ_, var_index } => {
-      use yatima_core::parse;
+    Cli::Show { typ: ShowType::Package { input } } => {
+      let store = Rc::new(FileStore::new());
+      match store.get(input) {
+        Some(ipld) => match yatima_core::package::Package::from_ipld(&ipld) {
+          Ok(pack) => println!("{}", pack),
```

Comment:
I'm not seeing such a utility in `core/parse.rs`, could you point it out? If its already defined it would be very easy to add to `yatima show package` in this pr if you want.

Reply:
Sorry, what I was referring to was the `Display` instance of `Defs` which is used by the `yatima parse` cli command. This just iterates through every `Def` in `Defs` and prints. I'd like to have something like this for `yatima show package`.

One problem with using the Defs Display for Packages though is that Defs are essentially maps, so they don't preserve key order. The better way to do it for `show package` (and also for `yatima parse`) is to get an iterator from the `Index` in the package (which has the same order that the defs were declared in the file) and using that to iterate over the Package Defs for printing. Does that make sense?

```rust
impl fmt::Display for Defs {
  fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
    for (k, v) in &self.names {
      let def = self.defs.get(v).unwrap();
      writeln!(f, "{}:", def.def_cid)?;
      writeln!(
        f,
        "def {} : {} = {}",
        k.clone(),
        def.typ_.pretty(Some(&k.to_string()), false),
        def.term.pretty(Some(&k.to_string()), false),
      )?;
      writeln!(f)?;
    }
    Ok(())
  }
}
```

So you 'd have something analogous to the above for e.g. a Display iml for Package, but iterating over `self.index.iter()`

Reply:
I think I get what you're saying, but how do we get the `defs` in the Display impl. Index just contains names and cids iirc.

Reply:
Well it doesn't have to literally be a `Display` impl, could just be a `-> String` function, but does Rust let you side effect inside `fmt`? If so we can convert the `index` to `Defs` with https://github.com/yatima-inc/yatima/blob/8661216b2371a75a78568f48ea696ae0be0da6c0/utils/src/file/parse.rs#L157. Can't do that in `core`, but in `utils` or the `cli` it's fine

Reply:
Hmm, okay, I'll see what I can do.

Reply:
Is there a way to get a `PackageEnv` easily from within the CLI @johnchandlerburnham If not I may let this wait for another pr.